src/utils/components/SimpleCopyButton/SimpleCopyButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, useState } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Button, clipboardCopyFunc, Tooltip } from '@patternfly/react-core';
+import CopyIcon from '@patternfly/react-icons/dist/esm/icons/copy-icon';
+
+import './SimpleCopyButton.scss';
+
+type SimpleCopyButtonProps = {
+  textToCopy: string;
+};
+
+const SimpleCopyButton: FC<SimpleCopyButtonProps> = ({ textToCopy }) => {
+  const { t } = useKubevirtTranslation();
+  const [copiedState, setCopiedState] = useState<boolean>(false);
+
+  const handleClick = (event: any) => {
+    clipboardCopyFunc(event, textToCopy);
+    setCopiedState(true);
+  };
+
+  return (
+    <Tooltip
+      aria="none"
+      aria-live="polite"
+      className="simple-copy-button"
+      content={copiedState ? <div>{t('Copied')}</div> : <div>{t('Copy to clipboard')}</div>}
+      entryDelay={300}
+      exitDelay={0}
+      maxWidth="150px"
+      onTooltipHidden={() => setCopiedState(false)}
+      position="top"
+      removeFindDomNode={false}
+      trigger="mouseenter focus click"
+    >
+      <Button
+        aria-label="Copy text to clipboard"
+        className="simple-copy-button__button"
+        icon={<CopyIcon />}
+        id="simple-copy-button__button"
```

Comment:
u are adding classname here, but on the scss u are using id selector, so no need for classname. i would give it a better id name if u are using it as selector to manipulate CSS as this can cause issue with styling in the future for other components. or change selector to classname but I also recommend on a better naming

Reply:
Fixed!